backtesting/backtesting.py

```diff
@@ -374,7 +376,8 @@ def __init__(self, broker: '_Broker',
                  stop_price: float = None,
                  sl_price: float = None,
                  tp_price: float = None,
-                 parent_trade: 'Trade' = None):
+                 parent_trade: 'Trade' = None,
+                 order_id = None):
```

Comment:
Let's call the parameter `tag=` because tags don't need to be unique.

Reply:
Great idea, renaming all to `tag` now...